src/components/JdsCalendar/Calendar.vue

```
@@ -0,0 +1,189 @@
+import JdsIcon from '../JdsIcon'
+
+export default {
+  name: 'jds-calendar',
+  components: {
+    JdsIcon
+  },
+  model: {
+    prop: 'value',
+    event: 'change',
+  },
+  props:{
+    /**
+     * The name for calendar
+     */
+    name: {
+      type: String
+    },
+    /**
+     * Type of calendar
+     * Value is `picker | range`
+     */
+    type: {
+     type: String,
+     default: 'picker'
+    },
+    /**
+     * You can set minimal date for
+     * the earliest acceptable date
+     * The value of the min date format follows the pattern `DD/MM/YYYY`.
+     */
+    min: {
+      type: String,
+      default: '01/01/1900'
+    },
+    /**
+     * You can set maximal date for
+     * the latest acceptable date
+     * The value of the max date format follows the pattern `DD/MM/YYYY`.
+     */
+    max: {
+      type: String,
+      default: '31/12/9999'
+    },
+    /**
+     * Bound model.
+     * @name value
+     * @model
+     * The value of the input date format follows the pattern `DD/MM/YYYY`.
+     * if value not set the value will be current date
+     */
+    value: {
+      type: String,
+    },
+  },
+  data() {
+    return {
+      mValue: undefined,
+      selectedValue: null,
+      listOfDays,
+      listOfMonth
+    }
+  },
+  watch: {
+    value: {
+      handler: function (v) {
+        if(typeof v === 'undefined' || v === null || !this.isDateStringValid(v)){
+          this.mValue = new Date()
+        }else{
+          this.mValue = fnDate.parseDate(v)
+          this.selectedValue = v.split('/')
+        }
+      },
+      immediate: true,
+    }
+  },
+  computed: {
+    days () {
+      return getDays(this.mValue)
+    }
+  },
+  methods: {
+    isDateStringValid(dateString){
+      let valid = true
+      const arrayOfDate = dateString.split("/")
+      const date = new Date(arrayOfDate[1]+"/"+arrayOfDate[0]+"/"+arrayOfDate[2])
+      if(date.toString() === 'Invalid Date'){
+        valid = false
+      }
+      return valid
```

It seems like this lines were the causes of issue mentioned by Shinta.

Due to Storybook Controls behavior, in which `value` is typed sequentially, there is a case such as this.
1) I want to entry '08/02/2000' as `DateInput#value`
2) I type each character into Storybook Controls
3) `isDateStringValid` will return false for each typed character, unless it reach `08/02/`, in which it return true

OTOH, `parseDate` function can't correctly parse `08/02/` as valid date.

---

This error can be ignored if `value` is not designed to accept string sequentially.